charts/victoria-logs-single/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: v0.28.0-victorialogs
+appVersion: v0.28.0
```

Comment:
VictoriaLogs releases have `-victorialogs` suffix in published docker images, see: https://hub.docker.com/r/victoriametrics/victoria-logs/tags?page=&page_size=&ordering=&name=v0.28.0

Reply:
using common template for image name generation, victorialogs was moved to `image.variant`, how it's done in the rest of the charts